app/api/api_v1/routers/search.py

```diff
@@ -245,28 +246,18 @@ def process_search_keyword_filters(
     db: Session,
     request_filters: Mapping[FilterField, Sequence[str]],
 ) -> Mapping[FilterField, Sequence[str]]:
+    filters = _convert_filters(db, request_filters)
+    if not filters:
+        return None
```

Comment:
Does the return type of the function need changing if the response can be None?

Reply:
Yep, thanks for flagging, I'll get it on the next sweep